lib/elixir_internal_certificate_web/controllers/api/v1/user_registration_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.UserRegistrationController do
+  use ElixirInternalCertificateWeb, :controller
+
+  alias ElixirInternalCertificate.Account.Accounts
+  alias ElixirInternalCertificate.Account.Schemas.User
+  alias ElixirInternalCertificateWeb.Api.ErrorView
+  alias ElixirInternalCertificateWeb.Api.V1.UserRegistrationView
+
+  def create(conn, params) do
+    with changeset <- Accounts.change_user_registration(%User{}, params),
+         {:ok, user} <- Accounts.register_user(changeset.changes) do
+      conn
+      |> put_view(UserRegistrationView)
```

Comment:
if we add the `UserRegistrationView` in `V1` module, we no need to use the `put_view` here

```
V1.UserRegistrationView
```